python/tvm/relay/frontend/tflite.py

```diff
@@ -2357,6 +2371,20 @@ def get_expr(self, input_tensor_idx):
     def has_expr(self, input_tensor_idx):
         return self.exp_tab.has_expr(get_tensor_name(self.subgraph, input_tensor_idx))
 
+    def get_tensor_or_const_expr(self, tensor):
```

Comment:
This name is trick. Here, we want to get relay expr and distinguish const. However, const is also expr in fact. Maybe we could name it `get_tensor_expr`? Do you have any better suggestion?